tapir.json

```
@@ -0,0 +1,15 @@
+{
+    "80002": [
+        "https://polygon-amoy-bor-rpc.publicnode.com",
+        "https://rpc-amoy.polygon.technology"
+    ],
```

Updated suggestion based on sorting.

```suggestion
    "80002": [
        "https://polygon.drpc.org",
        "https://polygon-amoy-bor-rpc.publicnode.com",
        "https://rpc-amoy.polygon.technology"
    ],
```